ocs_ci/ocs/precheck.py

```diff
@@ -0,0 +1,25 @@
+import logging
+from ocs_ci.ocs.resources.pod import get_osd_pods
+
+log = logging.getLogger(__name__)
+
+
+def check_max_osds_per_node(max_number_osds=3):
+    """
```

Comment:
If I see max_number_osds I would think of this param it is max allowed value. I know you are exlainin in doctring but what about to change'    if count >= max_number_osds:' to '    if count > max_number_osds:' And update doc to reflect this?

Reply:
Or change the docstring ?
```suggestion
    Function to check there is not more than `max_number_osds` per node
```
```suggestion
    Function to check there is less than or equal to `max_number_osds` per node
```

Reply:
Going with count > max_numer_ods, makes more sense that way (I couldn't come with reasonable variable name for '>=' condition)